src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -89,16 +89,20 @@
                 </p>
 
                 <div v-if="!isSummaryView && orgPerson.actions">
-                  <v-chip v-if="wasRemoved(orgPerson)" x-small label color="#grey lighten-2">
-                    {{ orgPerson.actions[0] }}
+                  <!-- show DELETED item as black on light gray -->
+                  <v-chip v-if="wasRemoved(orgPerson)" x-small label color="grey lighten-2">
+                    REMOVED
                   </v-chip>
-                  <v-chip
-                    v-else
-                    v-for="(action, i) in orgPerson.actions"
-                    :key="`action-chip-${i}`"
-                    x-small label color="primary" text-color="white">
-                    {{ action }}
+                  <!-- show REPLACED item separately (since item also has ADDED action) -->
+                  <v-chip v-else-if="wasReplaced(orgPerson)" x-small label color="primary" text-color="white">
+                    CHANGED
                   </v-chip>
+                  <!-- all other states -->
+                  <template v-else v-for="(action, i) in orgPerson.actions">
+                    <v-chip x-small label color="primary" text-color="white" :key="`action-chip-${i}`">
+                      {{ action }}
+                    </v-chip>
+                  </template>
```

Comment:
I broke this up to
1. be able to handle replaced/changed separately
2. make it easier to understand the various scenarios